### PR TITLE
FIX: multiple fixes for the release workflow

### PR DESCRIPTION
**Pull request by DBCerigo**
WARN: matching doesn't use regex (😒), it uses its own filter
syntax.

### CODE REVIEW

**Review by ali-tny — APPROVED** [review]

**Comment by ali-tny** [comment]
can you change the name of the PR please?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        if [ -f requirements.txt ]; then make install; fi
+        then make install
```
Comment (ali-tny) [inline .github/workflows/release.yml:25]:
```suggestion
        make install
```

**Review by ali-tny — CHANGES_REQUESTED** [review]
great try! just a small thing

**Review by ali-tny — APPROVED** [review]
nice

File: .github/workflows/release.yml (hunk 1 above)
Comment (DBCerigo) [reply .github/workflows/release.yml]: f u

**Pull request merged**